src/Tools/IdeCoreBenchmarks/FindReferencesBenchmarks.cs

```diff
@@ -56,11 +56,11 @@ public void Setup()
 
             // Force a storage instance to be created.  This makes it simple to go examine it prior to any operations we
             // perform, including seeing how big the initial string table is.
-            var storageService = (IPersistentStorageService2)_workspace.Services.GetService<IPersistentStorageService>();
+            var storageService = _workspace.Services.GetService<IPersistentStorageService>();
             if (storageService == null)
                 throw new ArgumentException("Couldn't get storage service");
 
-            using var storage = storageService.GetStorageAsync(_workspace.CurrentSolution, CancellationToken.None).GetAwaiter().GetResult();
+            using var storage = storageService.GetStorageAsync(_workspace.CurrentSolution, CancellationToken.None).AsTask().GetAwaiter().GetResult();
```

Comment:
💡 You can make `Setup()` asynchronous provided we are using BenchmarkDotNet 0.11.2 (https://github.com/dotnet/BenchmarkDotNet/issues/521)

Reply:
oh nice.  thanks!